libusb/os/windows_winusb.c

```diff
@@ -722,10 +723,50 @@ static void winusb_exit(struct libusb_context *ctx)
 	exit_dlls();
 }
 
+static BOOL device_io_control(HANDLE event, HANDLE hDevice, DWORD dwIoControlCode, LPVOID lpInBuffer, DWORD nInBufferSize, LPVOID lpOutBuffer, DWORD nOutBufferSize, LPDWORD lpBytesReturned)
+{
+	OVERLAPPED overlapped;
+	DWORD error;
+	BOOL ret = TRUE;
+
+	ResetEvent(event);
+	SecureZeroMemory(&overlapped, sizeof(overlapped));
+	overlapped.hEvent = event;
+
+	if (DeviceIoControl(hDevice, dwIoControlCode, lpInBuffer, nInBufferSize, lpOutBuffer, nOutBufferSize, lpBytesReturned, &overlapped)) goto end;
+
+	error = GetLastError();
+	if (error != ERROR_IO_PENDING) {
+		CancelIo(hDevice);
```

Comment:
shouldn't this be:
```suggestion
		CancelIoEx(hDevice, &overlapped);
```
?